desktop_version/src/editor.cpp

```diff
@@ -2642,6 +2642,12 @@ void editorrender()
 
     int temp2=edentat(ed.tilex+ (ed.levx*40),ed.tiley+ (ed.levy*30));
 
+    // Special case for drawing gray entities
+    int current_room = (ed.levx+(ed.levy*ed.maxwidth));
+    bool custom_gray = ed.level[current_room].tileset == 3 && ed.level[current_room].tilecol == 6;
```

Comment:
Should probably bounds-check this index of `ed.level`.
```suggestion
    bool custom_gray = INBOUNDS_ARR(current_room, ed.level)
    && ed.level[current_room].tileset == 3 && ed.level[current_room].tilecol == 6;
```
Same could be said for `graphics.entcolours` below, too, but that code already didn't have a bounds check, so I addressed that in #476.